libs/payments/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "module": "commonjs"
   },
+  "target": "ES2018",
```

Comment:
Should be in the tsconfig.lib.json and probably be es2021 looking at the other libraries